modules/nodes-containers-sysctls-unsafe.adoc

```diff
@@ -72,58 +73,85 @@ spec:
 ----
 $ oc apply -f set-sysctl-worker.yaml
 ----
-+
-A new `MachineConfig` object named in the `99-worker-XXXXXX-XXXXX-XXXX-XXXXX-kubelet` format is created.
 
-. Wait for the cluster to reboot usng the `machineconfigpool` object `status` fields:
+. Wait for the Machine Config Operator to generate the new rendered configuration and apply it to the machines:
 +
-For example:
+[source,terminal]
+----
+$ oc get machineconfigpool worker -w
+----
++
+After some minutes the `UPDATING` status changes from True to False:
++
+[source,terminal]
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+worker   rendered-worker-f1704a00fc6f30d3a7de9a15fd68a800   False     True       False      3              2                   2                     0                      71m
+worker   rendered-worker-f1704a00fc6f30d3a7de9a15fd68a800   False     True       False      3              2                   3                     0                      72m
+worker   rendered-worker-0188658afe1f3a183ec8c4f14186f4d5   True      False      False      3              3                   3                     0                      72m
+----
+. Create a YAML file `sysctl-example-safe-unsafe.yaml` that defines an example pod and add the `securityContext` spec, as shown in the following example:
 +
 [source,yaml]
 ----
-status:
-  conditions:
-    - lastTransitionTime: '2019-08-11T15:32:00Z'
-      message: >-
-        All nodes are updating to
-        rendered-worker-ccbfb5d2838d65013ab36300b7b3dc13
-      reason: ''
-      status: 'True'
-      type: Updating
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sysctl-example-safe-unsafe
+spec:
+  containers:
+  - name: podexample
+    image: centos
+    command: ["bin/bash", "-c", "sleep INF"]
+  securityContext:
+    sysctls:
+    - name: kernel.shm_rmid_forced
+      value: "0"
+    - name: net.core.somaxconn
+      value: "1024"
+    - name: kernel.msgmax
+      value: "65536"
 ----
+
+. Create the pod:
 +
-A message similar to the following appears when the cluster is ready:
+[source,terminal]
+----
+$ oc apply -f sysctl-example-safe-unsafe.yaml
```

Comment:
After MachineConfig update the warning message appears anyway
`W0616 21:03:08.151334 1404026 warnings.go:70] would violate PodSecurity "restricted:latest": forbidden sysctls (net.core.somaxconn, kernel.msgmax)`